boards/shared/Src/App/App_SharedErrorTable.c

```diff
@@ -69,14 +69,16 @@ struct ErrorTable *App_SharedErrorTable_Create(void)
     INIT_ERROR(PDM_NON_CRITICAL_WATCHDOG_TIMEOUT, PDM, NON_CRITICAL_ERROR);
 
     INIT_ERROR(BMS_AIR_SHUTDOWN_CHARGER_DISCONNECTED_IN_CHARGE_STATE, BMS, AIR_SHUTDOWN_ERROR);
-    INIT_ERROR(BMS_AIR_SHUTDOWN_MAX_CELL_VOLTAGE_OUT_OF_RANGE, BMS, AIR_SHUTDOWN_ERROR);
-    INIT_ERROR(BMS_AIR_SHUTDOWN_MIN_CELL_VOLTAGE_OUT_OF_RANGE, BMS, AIR_SHUTDOWN_ERROR);
-    INIT_ERROR(BMS_AIR_SHUTDOWN_HAS_PEC_ERROR, BMS, AIR_SHUTDOWN_ERROR);
-    INIT_ERROR(BMS_AIR_SHUTDOWN_MIN_CELL_TEMP_OUT_OF_RANGE, BMS, AIR_SHUTDOWN_ERROR);
-    INIT_ERROR(BMS_AIR_SHUTDOWN_MAX_CELL_TEMP_OUT_OF_RANGE, BMS, AIR_SHUTDOWN_ERROR);
+    INIT_ERROR(BMS_AIR_SHUTDOWN_CELL_OVERVOLTAGE_FAULT, BMS, AIR_SHUTDOWN_ERROR);
+    INIT_ERROR(BMS_AIR_SHUTDOWN_CELL_UNDERVOLTAGE_FAULT, BMS, AIR_SHUTDOWN_ERROR);
+    INIT_ERROR(BMS_AIR_SHUTDOWN_HAS_PEC_FAULT, BMS, AIR_SHUTDOWN_ERROR);
+    INIT_ERROR(BMS_AIR_SHUTDOWN_CELL_UNDERTEMP_FAULT, BMS, AIR_SHUTDOWN_ERROR);
+    INIT_ERROR(BMS_AIR_SHUTDOWN_CELL_OVERTEMP_FAULT, BMS, AIR_SHUTDOWN_ERROR);
     INIT_ERROR(BMS_AIR_SHUTDOWN_CHARGER_FAULT_DETECTED, BMS, AIR_SHUTDOWN_ERROR);
     INIT_ERROR(BMS_AIR_SHUTDOWN_HAS_REACHED_MAX_V, BMS, AIR_SHUTDOWN_ERROR);
     INIT_ERROR(BMS_AIR_SHUTDOWN_CHARGING_EXT_SHUTDOWN_OCCURRED, BMS, AIR_SHUTDOWN_ERROR);
+    INIT_ERROR(BMS_AIR_SHUTDOWN_TS_OVERCURRENT_FAULT, BMS, AIR_SHUTDOWN_ERROR);
```

Comment:
Updated SharedErrorTable with new faults, as well as updated names of existing faults, however the plan is still to phase the table out, just not in this PR